pkg/exporter/syslogexporter/factory_test.go

```diff
@@ -41,14 +40,16 @@ func TestCreateDefaultConfig(t *testing.T) {
 		Format:   "rfc5424",
 		QueueSettings: exporterhelper.QueueSettings{
 			Enabled:      false,
-			NumConsumers: 0,
-			QueueSize:    0,
-			StorageID:    (*component.ID)(nil)},
+			NumConsumers: 10,
+			QueueSize:    5000,
```

Comment:
I think this is the wrong default and should be kept much lower. With a batch size of 1000, this would eat several GB of memory, which is definitely excessive. A value like 50 or even 10 is much more reasonable imo.

Reply:
These default settings comes from exporterhelper https://github.com/SumoLogic/sumologic-otel-collector/blob/df9f06125b8e79aaeae3a0636519e4d64f606591/pkg/exporter/syslogexporter/factory.go#L42-L43 and we have the same settings in sumologic exporter https://github.com/SumoLogic/sumologic-otel-collector/blob/df9f06125b8e79aaeae3a0636519e4d64f606591/pkg/exporter/sumologicexporter/factory.go#L44-L45

Reply:
I know, and I think those are bad defaults there as well. I'm planning to open a ticket upstream about it.

Reply:
so I would prefer to have this fixed in upstream rather in this exporter, when it will be fixed in upstream it will be fixed for this exporter also after updating dependency

Reply:
Here's the upstream issue: https://github.com/open-telemetry/opentelemetry-collector/issues/7359